src/cr/cube/matrix/measure.py

```diff
@@ -427,9 +427,9 @@ def smoothed_column_proportions(self):
         object will be returned, otherwise fallback to unsmoothed measure object.
         """
         column_dimension = self._dimensions[-1]
-        smoothing_spec = column_dimension.smoothing_spec
+        smoother = BaseSmoothingSpec.smoother(column_dimension)
```

Comment:
we get the smoother directly from the `BaseSmoothingSpec` and pass it to the measure class